### PR TITLE
Libpitt/427 mouse meta

### PR DESCRIPTION
**Pull request by libpitt**
Change log:
1. Add support for Mouse metadata
2. Update logic of metadata notice

### CODE REVIEW

File: src/pages/edit/sample.jsx
Comment (maxsibilla) [inline src/pages/edit/sample.jsx:1]: What is the reason to pull this text component out of the IF statement but not the one on line 373?

Comment (libpitt) [reply src/pages/edit/sample.jsx:1]: The one on line 373 is for whenever there's a change of specimen type when metadata exists.

Comment (maxsibilla) [reply src/pages/edit/sample.jsx:1]: So shouldn't `text.push(getMetadataNote(cache.entities.sample, 1))` remain inside
```
if (data.sample_category === values.sample_category) {
```

Comment (libpitt) [reply src/pages/edit/sample.jsx:1]: Not necessarily. The case 1  is "You may view it via [the full entity JSON](http://localhost:3000/api/json/sample?uuid=d3ff9682d4550e3a890527679d62e615)."
And a user may want to see what they are changing while updating the specimen type.

Screen Shot 2023-06-08 at 2 41 49 PM

Case 2 is "Changing the ..."

**Pull request merged**